src/main/java/pl/agawesolowska/ticketbookingapp/model/TicketType.java

```diff
@@ -0,0 +1,57 @@
+package pl.agawesolowska.ticketbookingapp.model;
+
+import java.math.BigDecimal;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Getter;
+
+/**
+ * @author Aga Wesołowska
+ *
+ */
+@Getter
+public enum TicketType {
+
+	@JsonProperty("adult")
+	ADULT(BigDecimal.valueOf(25)),
+	
+	@JsonProperty("student")
+	STUDENT(BigDecimal.valueOf(18)),
+	
+	@JsonProperty("child")
+	CHILD(BigDecimal.valueOf(12.50));
+
+	private BigDecimal price;
+
+	private TicketType(BigDecimal price) {
+		this.price = price;
+	}
+
+	public void setWeekendPrice(TicketType type) {
+		switch (type) {
+		case ADULT:
+			this.price = BigDecimal.valueOf(29);
+			break;
+		case STUDENT:
+			this.price = BigDecimal.valueOf(22);
+			break;
+		case CHILD:
+			this.price = BigDecimal.valueOf(16.50);
+			break;
+		}
+	}
+
+	public String toString(TicketType type) {
+		switch (type) {
+		case ADULT:
+			return "25 PLN";
```

Comment:
dlaczego to jest wpisane na sztywno? a co jak zmienisz wartości w enumie a tu nie?

Reply:
Ta metoda powstała w odpowiedzi na wymagania - There are three ticket types: adult (25 PLN), student (18 PLN), child (12.50 PLN) - zaznaczyliście walutę, dlatego chciałam jakoś z tego wybrnąć :) Nie jest ona niezbędna i z tego co pamiętam w komponencie Angulara można chyba ustawić walutę, ale zgodzę się że to błąd, jest wpisane na sztywno.